src/components/Button.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import '../App.css';
 
 function Button(props) {
```

Comment:
[optional] It's best to use `es6` arrow functions in react.js rather than the `es5` `function` syntax except when you have to this helps to stabilize this value of `this` across all scopes.